lastminutewf2020.yml

```diff
@@ -1,5 +1,36 @@
 ---
 - hosts: all
+  tasks:
+     - name: Pre-generate the kernel flags for ansible usage
+       set_fact:
+         procline: "quiet ipv6.disable=1 cgroup_enable=memory swapaccount=1"
+    - name: Ensure /etc/default/grub has proper GRUB_CMDLINE_LINUX_DEFAULT
+      ansible.builtin.replace:
+        path: /etc/default/grub
+        regexp: '^GRUB_CMDLINE_LINUX_DEFAULT='
+        line: "GRUB_CMDLINE_LINUX_DEFAULT=\"{{ procline }}\""
+        owner: root
+        group: root
+        mode: '0644'
+      notify:
+        -  update grub
+        -  reboot
+    - name: Ensure /etc/default/grub has proper GRUB_TIMEOUT
+      ansible.builtin.replace:
+        path: /etc/default/grub
+        regexp: "GRUB_TIMEOUT=.*"
+        replace: "GRUB_TIMEOUT=3"
+        owner: root
+        group: root
+        mode: '0644'
+      notify:
+        -  update grub
+        -  reboot
```

Comment:
nitpick, this one does not need the reboot as it only affects grub and not the kernel so its only relevant when the machine restarts and at that point it will read it.